app/services/idv/usps_validator.rb

```diff
@@ -0,0 +1,15 @@
+module Idv
+  class UspsValidator < VendorValidator
```

Comment:
codeclimate says this is completely untested, we should at the very least have features spec that clicks through the pages this thing does if we can't make unit tests for it.